docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -6,53 +6,27 @@ group: Deploying Alluxio
 priority: 3
 ---
 
-TODO: Focus on tmpfs + domain sockets to avoid issues with memory limits.
-TODO: Restructure to be similar to other deploy docs.
-
-Introduction: Give more context/examples about why someone would want to run Alluxio in
-a Docker container.
-
-1. Prerequisites: Generalize the current prerequisites so that they can apply to real
-production environments.
-2. Launch Alluxio
-- Tell the user how to run from dockerhub (minimal example w/ 1 worker 1 master)
-3. Verify the Cluster
-- runTests, check the web UI, run fsadmin report
-4. Operations (Only include if different from standalone cluster deployment)
-- How to set server configuration
-- How to use domain sockets for short-circuit read/write
-- How to stop or restart Alluxio
-- How to format the journal
-- How to upgrade Alluxio to a newer version
-- How to add/remove workers/masters
-- How to run in HA mode
-- How to tweak and rebuild the docker image
-6. Troubleshooting
-- How to find logs
-- Where to get help
-
 * Table of Contents
 {:toc}
 
-Alluxio can be run in a Docker container. This guide demonstrates how to run Alluxio
-in Docker using the Dockerfile provided in the distribution.
-
-## Basic Tutorial
-
-This tutorial walks through a basic dockerized Alluxio setup on a single node.
-
-### Prerequisites
+Docker can be used to simplify the deployment and management of Alluxio servers.
+Using the `alluxio/alluxio` Docker image available on Dockerhub, you can go from 
+zero to a running Alluxio cluster in just a couple of `docker run` commands.
+This document provides a tutorial for running Dockerized Alluxio on a single node. 
+We'll also discuss more advanced topics, and how to troubleshoot.
 
-A Linux machine. For the purposes of this guide, we will use a fresh EC2 machine running
-Amazon Linux. The machine size doesn't need to be large; we will use t2.small.
-When setting up the network security for the instance, allow traffic on ports 19998-19999
-and 29998-30000.
+## Prerequisites
 
-### Launch a standalone cluster
+- A Linux machine with Docker installed.
+- Ports 19998, 19999, 29998, 29999, and 30000 are available
```

Comment:
nit: it would be nice to have a single page for default port requirements linked to by all doc pages

Reply:
deferring this to a later PR